### PR TITLE
fix(pdp): align env template with config and remove Asgardeo-specific IDP_ORG_NAME

### PR DESCRIPTION
**Pull request by mushrafmim**
## Summary

Brings the Policy Decision Point (PDP) environment configuration in line with what the code actually reads, and removes the Asgardeo-specific `IDP_ORG_NAME` in favor of explicit, provider-agnostic OIDC settings. Documentation-and-config only; no behavioral change to request handling.

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update
- [x] Refactoring (no functional changes)
- [ ] Performance improvement
- [ ] Other (please describe):

## Changes Made

- **`.env.template`**: now matches the variables the config loader actually reads.
  - Removed variables the code never reads: `IDP_CLIENT_ID`, `IDP_CLIENT_SECRET`, `LOG_LEVEL` (log level is a flag defaulted from `ENVIRONMENT`, not an env var), and the duplicate `PORT`.
  - Added variables the code reads but were missing: `IDP_AUDIENCE`, `IDP_JWKS_URL`.
  - Removed `IDP_ORG_NAME` (Asgardeo-specific, not an OIDC concept) and relabeled the section as OIDC.
- **`internal/config/config.go`**: dropped `IDP_ORG_NAME`/`OrgName` (the `IDPConfig.OrgName` field, the `orgName` lookup, and the struct assignment); default `IDP_ISSUER`/`IDP_JWKS_URL` are now generic (`https://localhost:8090/oauth2/...`) instead of org-templated Asgardeo URLs.
- **`main.go`**: removed the now-defunct `org_name` log field.
- **`README.md`**: minor services-table formatting tidy.

## Testing

- [x] I have tested this change locally
- [ ] I have added unit tests for new functionality
- [ ] I have tested edge cases
- [x] All existing tests pass

- `go build ./...` / `go vet ./...` clean for PDP.
- Verified against `internal/config/config.go` that the template variables are exactly the set consumed by `LoadConfig` (plus `RUN_MIGRATION` from `v1/database.go`).

## Checklist

- [x] My code follows the project's style guidelines
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have checked that there are no merge conflicts

## Additional Notes

- The PDP IDP config is currently loaded and logged only (not yet consumed by JWT verification), so these changes are safe and forward-looking.
- Follow-ups intentionally left out of this PR: aligning `exchange/docker-compose.yml` PDP env (and possibly adding a Postgres service), and a fail-fast `GetRequiredEnv`-style helper for genuinely-required variables.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: exchange/policy-decision-point/internal/config/config.go [hunk 1]
```diff
@@ -75,10 +74,9 @@ func LoadConfig(serviceName string) *Config {
 	flag.Parse()
 
 	// Reading IDP Configs
-	orgName := utils.GetEnvOrDefault("IDP_ORG_NAME", "YOUR_ORG_NAME")
-	userIssuer := utils.GetEnvOrDefault("IDP_ISSUER", "https://api.asgardeo.io/t/"+orgName+"/oauth2/token")
+	userIssuer := utils.GetEnvOrDefault("IDP_ISSUER", "https://localhost:8090/oauth2/token")
```
[the remark this answers is not in the document]
Comment (mushrafmim) [reply exchange/policy-decision-point/internal/config/config.go]: Addressed

**Review by ginaxu1 — APPROVED** [review]
